openreview/venue/process/sac_paper_assignment_pre_process.js

```diff
@@ -0,0 +1,21 @@
+async function process(client, edge, invitation) {
```

Comment:
can we just use the same assignment_pre_process that we use for the other groups?

Reply:
done